packages/eslint-plugin-svelte/src/rules/no-add-event-listener.ts

```diff
@@ -0,0 +1,74 @@
+import type { TSESTree } from '@typescript-eslint/types';
+
+import { createRule } from '../utils/index.js';
+import type { SuggestionReportDescriptor } from '../types.js';
+
+export default createRule('no-add-event-listener', {
+	meta: {
+		docs: {
+			description: 'Warns against the use of `addEventListener`',
+			category: 'Best Practices',
+			recommended: false,
+			default: 'warn'
```

Comment:
Why was the default set to warn?

Reply:
some rules set this and some don't. i was trying to be consistent with the other ones, but have removed it since most don't specify this